extensions/amp-addthis/0.1/addthis-utils/pixel.js

```diff
@@ -201,13 +201,16 @@ export const callPixelEndpoint = event => {
       credentials: 'include',
     })
     .then(res => res.json())
-    .then(json => {
-      const {pixels = []} = json;
-      if (pixels.length > 0) {
-        dropPixelGroups(pixels, {
-          sid: eventData['sid'],
-          ampDoc,
-        });
-      }
-    });
+    .then(
+      json => {
+        const {pixels = []} = json;
+        if (pixels.length > 0) {
+          dropPixelGroups(pixels, {
+            sid: eventData['sid'],
+            ampDoc,
+          });
+        }
+      },
+      () => {}
```

Comment:
Do we want to send a user error?

Reply:
It's a server error, I'm not sure it'd be useful.

Reply:
I see. Agree